core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/QueryResponsePostProcessorImpl.java

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (c) Codice Foundation
+ * 
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * 
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ * 
+ **/
+package ddf.catalog.impl;
+
+import java.net.URISyntaxException;
+import java.net.URL;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import ddf.action.Action;
+import ddf.action.ActionProvider;
+import ddf.catalog.CatalogFramework;
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.Result;
+import ddf.catalog.data.impl.AttributeImpl;
+import ddf.catalog.operation.QueryResponse;
+import ddf.catalog.operation.SourceResponse;
+
+/**
+ * {@link CatalogFramework} proxy class used to transform the URIs found in a {@link SourceResponse}
+ * {@link Metacard}s using the injected {@link ActionProvider} before transformation by the
+ * {@link CatalogFramework}.
+ */
+public class QueryResponsePostProcessorImpl implements QueryResponsePostProcessor {
+    // PreCatalogTransformPlugin {
+    private static final Logger LOGGER = LoggerFactory
+            .getLogger(QueryResponsePostProcessorImpl.class);
+
+    private ActionProvider resourceActionProvider;
+
+    public QueryResponsePostProcessorImpl() {
```

Comment:
Is this ever used? Seems like this is kind of useless without the resourceActionProvider